Makefile

```diff
@@ -49,7 +49,7 @@ install-tools:
 	dotnet new tool-manifest || exit 0
 	dotnet tool install --local security-scan --version 5.6.3 || exit 0
 	dotnet tool install --local dotnet-format || exit 0
-	dotnet tool update -g docfx || exit 0
+	dotnet tool update --local docfx || exit 0
```

Comment:
Why update when the others are install?

Reply:
🤷🏻  Was just following the quick-start guide from DocFX. I can make it install since this realistically is a one-time step. Just means if you run it in the future, it won't update your existing installation.

Reply:
Install is ideal since the Makefile target is install. Updating would be a side-effect and we never want those.